setup.py

```diff
@@ -7,7 +7,7 @@
     url="https://github.com/CNugteren/pypopquiz",
     description="Python tool to create pop-quiz videos programmatically",
     license="MIT",
-    install_requires=["ffmpeg-python", "pytube", "jsonschema"],
+    install_requires=["ffmpeg-python", "pytube", "jsonschema", "moviepy", "requests"],
```

Comment:
Where is `requests` used actually?

Reply:
It's a workaround: moviepy uses "requests", but I suspect it doesn't list it in its setup.py. This caused one of the travis tests to fail (locally it often works because "requests" is quite common. There might be a nicer way to solve this that I am not aware of.